Bindings/Java/Matlab/tests/testConnectorsInputsOutputs.m

```diff
@@ -43,7 +43,7 @@
 end
 
 % Access (and iterate through) a component's connectees as objects.
-expectedNames = {'ground', 'block'};
+expectedNames = {'block', 'ground'};
```

Comment:
> If the latter are we guaranteed uniqueness?

Ah...there is an issue here. I will add a check to see if a connector/input with the same name already exists, and throw an exception if so.  Related: #859.